src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $width-card;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 300ms;
```

Comment:
Not critical, but it's better to specify the exact properties instead of `all`. Also, would be great to create variables for transition-duration and all colors
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x